packages/checkout-ui-extensions/src/components/Image/Image.ts

```diff
@@ -1,16 +1,19 @@
 import {createRemoteComponent} from '@remote-ui/core';
 
+import {Responsive} from '../shared';
+
 export interface ImageProps {
   /**
    * The default image path. This is used for single images, as a fallback if
    * no sources match, or if a browser does not support picture element sources.
    */
   source: string;
   /**
-   * An array of image sources to offer alternative versions of an image for
-   * different device widths and pixel densities
+   * An object where the keys are the device widths and the values are arrays of
+   * image sources to offer alternative versions of an image for
+   * different device widths and pixel densities.
    */
-  sources?: Source[];
+  sources?: Responsive<string | Source | Source[]>;
```

Comment:
@jffortier @kumar303 do you think we have enough consumers and enough breaking changes to warrant an upgrade guide?

Reply:
I might be wrong, but we don't have any consumers yet. Partners are using the post-purchase version of these components.

Reply:
Changelog issue created in checkout-web
https://github.com/Shopify/checkout-web/issues/5785